var/spack/repos/builtin/packages/py-flash-attn/package.py

```diff
@@ -11,17 +11,30 @@ class PyFlashAttn(PythonPackage):
     This package provides the official implementation of FlashAttention.
     """
 
-    pypi = "flash-attn/flash_attn-2.5.4.tar.gz"
+    homepage = "https://github.com/Dao-AILab/flash-attention.git"
+    pypi = "flash-attn/flash_attn-0.0.0.tar.gz"
+    git = "https://github.com/Dao-AILab/flash-attention.git"
 
     maintainers("aurianer")
 
     license("BSD")
 
+    version("main", branch="main")
+    version(
+        "2.5.9.post1", sha256="a92db1683a5b141a0f4371d251ae9f73e9aef629b3a58a50d0ef430266c68782"
```

Comment:
Do you know what the deal is with this `post1` version? 2.5.9 exists but doesn't have all the same assets on GitHub so I assume this is just a weird flash-attn quirk? Not a blocker, just thought I'd check before this is merged.

Reply:
Thanks. If you find out later that 2.5.9.post1 is straight up broken, feel free to mark it deprecated next time you update the package (if there is a next time).